security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -174,8 +173,8 @@ func (s *sdsservice) DebugInfo() (string, error) {
 			ProxyID:      conn.proxyID,
 			ResourceName: conn.ResourceName,
 			// should be base64 encoded for straightforward comparison with Envoy SDS config dump format
```

Comment:
Could you update the comment here as well?

Reply:
updated 👍